docs/brief-install-vagrant-dev.md

```diff
@@ -105,3 +105,15 @@ HOST_PORT 9971
 
 (and halt and restart the Vagrant guest), then you would visit
 http://localhost:9971/ to connect to your development server.
+
+If you wish to set a different host IP address for Vagrant to use
+rather than localhost (127.0.0.1), you can add that to your
+'~/.zulip-vagrant-config' file as well. For example, if you set:
+
+```
+HOST_IP_ADDR 0.0.0.0
+```
+
+(and restart the Vagrant guest), your host IP would be 0.0.0.0, a
+special host IP address that would allow other computers to connect
+to your development server.
```

Comment:
Maybe describe it as "a special value for the IP address that means any IP address can connect to your development server"?